.github/workflows/windows.yml

```diff
@@ -0,0 +1,51 @@
+name: Windows build
+
+on: [push, pull_request]
+
+env:
+  # Customize the env if
+  BUILD_TYPE: Release
+  DPDK_VERSION: 22.07
+
+jobs:
+  Build:
+    if: github.repository == 'OpenVisualCloud/Media-Transport-Library'
+    runs-on: [Windows, self-hosted]
+
+    steps:
+    - name: checkout kahawai repo
+      uses: actions/checkout@v3
+      with:
+        path: kahawai
+    
+    - name: checkout dpdk repo
+      uses: actions/checkout@v3
+      with:
+        repository: 'DPDK/dpdk'
+        ref: v${{  env.DPDK_VERSION  }}
+        path: dpdk
+
+    - name: Apply patches
+      run: |
+        cd dpdk
+        git am ../kahawai/patches/dpdk/${{  env.DPDK_VERSION  }}/*.patch
+        git am ../kahawai/patches/dpdk/${{  env.DPDK_VERSION  }}/windows/*.patch
+      shell: bash
+      
+    - name: Build dpdk
+      run: |
+        cd dpdk
+        meson build --prefix=C:\dpdk
+        ninja -C build install
+        cd c:\dpdk\lib
+        rm *.dll.a
+        (gc "C:\dpdk\lib\pkgconfig\libdpdk.pc") | ForEach-Object {$_ -replace('Libs.private: .+','Libs.private:')} | Out-File -Encoding  ASCII "C:\dpdk\lib\pkgconfig\libdpdk.pc"
+        (gc "C:\dpdk\lib\pkgconfig\libdpdk-libs.pc") | ForEach-Object {$_ -replace('Libs: .+','Libs:-L${libdir} -lrte_latencystats -lrte_gso -lrte_bus_pci -lrte_gro -lrte_cfgfile -lrte_bitratestats -lrte_timer -lrte_hash -lrte_metrics -lrte_cmdline -lrte_pci -lrte_ethdev -lrte_meter -lrte_net -lrte_net_ice -lrte_net_iavf -lrte_common_iavf -lrte_mbuf -lrte_mempool -lrte_stack -lrte_mempool_stack -lrte_mempool_ring -lrte_rcu -lrte_ring -lrte_eal -lrte_telemetry -lrte_kvargs -lrte_dmadev -lrte_dma_ioat')} | Out-File -Encoding ASCII "C:\dpdk\lib\pkgconfig\libdpdk-libs.pc"
+
+    - name: Build kahawai
+      env:
+        PKG_CONFIG_PATH: 'C:\dpdk\lib\pkgconfig;c:\libst_dpdk\lib\pkgconfig;C:\kahawai\mingw-w64-x86_64-gtest-1.11.0-6\lib\pkgconfig;C:\kahawai\mingw-w64-x86_64-openssl-1.1.1.o-3\lib\pkgconfig;C:\kahawai\mingw-w64-x86_64-json-c-0.16-1\lib\pkgconfig;C:\kahawai\SDL2-2.0.22\x86_64-w64-mingw32\lib\pkgconfig'
```

Comment:
Can you try to rename libst_dpdk to libmtl since our official pkg name change to libmtl recently.